src/schema/objects/columns.yaml

```diff
@@ -48,6 +48,14 @@ cardiac:
   description: |
     continuous pulse measurement
   type: number
+channel:
+  name: channel
+  display_name: Channel
+  description: |
+    Channel(s) associated with an event.
+    If multiple channels are specified, separate with commas (`,`) or a
```

Comment:
This should be revised if the Delimiter column is now required (which I think we agreed it should be)?